content/blog/2020-04-27-moving-to-the-matrix/index.md

```diff
@@ -0,0 +1,185 @@
+  and the French government, to name a few.
+- Matrix instances can connect to each other ("federation"). This means from our
+  GNU Radio chat, we can access other organizations on Matrix, even if they're
+  not part of the GNU Radio chat instance. This includes the main Matrix instance
+  on [matrix.org](https://matrix.org), where projects such as our friends from
+  SatNOGS are located.
+- That means that people already on Matrix don't have to do a whole lot to join
+  our Matrix server.
+- Bridging to IRC is also easy, and will have a high likelihood of working
+  indefinitely.
+- The open standards nature of Matrix, and the availability of open source
+  reference clients, means that there are lots of clients out there, including
+  things like a [Weechat plugin](https://github.com/poljar/weechat-matrix).
+
+
+## The matter of Transparency, or why am I only hearing about this now?
+
+While the GNU Radio leadership team definitely did their due diligence in
+picking a new service, one thing that definitely shouldn't become the norm for
+decisions around the GNU Radio project is the fact that we decided this behind
+closed doors within the leadership team. For example, we did not publish a GREP
+ahead of time for a public discussion.
+
+After extensive discussions, we agreed that without actually having tried it,
+the decision for a chat service would almost exclusively be a battle of
+opinions. Having all reasonable opinions among us, we decided to forgo
+fragmenting the community and instead conducted a closed testing of the most
+promising candidates.
+
+## What happens to the GNU Radio Slack?
+
+We will turn off Slack end of August 2020.
+The main development, documentation, and variety of other discussions regarding
+the project will be held on our
+Matrix instance going forward, in order to preserve them in history.
+We are paying for a one-time unlocking of the Slack history, and will make
+them available in some form to all participants of the GNU Radio chat.
+
+Many people from the GNU Radio leadership will be leaving the Slack server, and
+won't be reachable there going forward, even before August.
+
+We could keep Slack running, since it's free, but we decided against it. This is
+for multiple reasons:
+- Even if administering the Slack instance is very little work, it is still work
+  and we don't want to have yet another task.
+- All GNU Radio leadership will move off of the Slack channel, so calling it a
+  GNU Radio chat service is a stretch anyway at that point.
+- Closing the service is the only way to guarantee that everyone on the GNU Radio
+  Slack definitely hears about the deprecation of our Slack.
+- We don't want to risk any GPDR issues by being responsible for other people's
+  data.
+
+## The new setup, and yes, we're still tuning it
+
+As of April 2020, we have rented a hosted Matrix setup at Swiss hosting company
+[unglei.ch](https://ungleich.ch/u/products/hosted-matrix-chat/). Our custom
```

Comment:
Also the name of the link should be [ungleich](https://ungleich.ch/u/products/hosted-matrix-chat/)